conf/logback.xml

```diff
@@ -16,7 +16,10 @@
     </appender>
 
     <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
-        <encoder class="net.logstash.logback.encoder.LogstashEncoder" />
+        <encoder>
+            <pattern>%date [%thread] %-5level %logger{36} - %msg%n%xException{3}</pattern>
+        </encoder>
+        <!--<encoder class="net.logstash.logback.encoder.LogstashEncoder" />-->
```

Comment:
❗ This needs changing back

Reply:
👍  c8966f2fb17e6c3377ab764028a0ee85d8984ca7